packages/newspringchurchapp/src/prayer/PrayerTabView/index.js

```diff
@@ -16,10 +12,6 @@ import GET_CAMPUS_PRAYERS from '../data/queries/getCampusPrayers';
 import GET_SAVED_PRAYERS from '../data/queries/getSavedPrayers';
 import PrayerTab from './PrayerTab';
 
-const StyledHorizontalTileFeed = styled({
```

Comment:
this may cause problems but for the life of me I can't figure out why it was here...